Add known issue on TopologyAwareHints for 1.21 (#1512)
kubernetes/sig-release: releases/release-1.21/release-notes-draft.md

The CSI health monitoring feature is being released as a second Alpha in Kubernetes 1.21. This feature enables CSI Drivers to share abnormal volume conditions from the underlying storage systems with Kubernetes so that they can be reported as events on PVCs or Pods. This feature serves as a stepping stone towards programmatic detection and resolution of individual volume health issues by Kubernetes.

## Known Issues

### `TopologyAwareHints` feature falls back to default behavior

> The feature gate currently falls back to the default behavior in most cases. Enabling the feature gate will add hints to `EndpointSlices`, but functional difference only takes place on non-dual stack kube-proxy implementation. [The fix will be available in coming releases](https://github.com/kubernetes/kubernetes/pull/100804).

Reviewer: ```suggestion
The feature gate currently falls back to the default behavior in most cases. Enabling the feature gate will add hints to `EndpointSlices`, but a functional difference is only observed in non-dual stack kube-proxy implementation. [The fix will be available in coming patch releases](https://github.com/kubernetes/kubernetes/pull/100804).
```

Author: 🤔 I agree with all except the "patch" part -- should we go that specific? since it should be available on _both_ 1.22 and 1.21.x (x > 0) releases right?

Author: what about this:
> ... but functional differences are only observed in non-dual stack kube-proxy

Reviewer: Your concern makes sense.

I was trying to be specific with the suggestion here. Let's keep it generic. `The fix will be available in coming releases`

Author: awesome done!
